packages/demo/src/App.svelte

```diff
@@ -79,8 +80,13 @@
   }
   const trezor = trezorModule(trezorOptions)
 
+  const magic = magicModule({
+    apiKey: 'pk_live_02207D744E81C2BA'
+  })
+
   const onboard = Onboard({
     wallets: [
+      magic,
```

Comment:
I think we can place this further down the list for the demo, maybe just in front of the Fortmatic module?